src/Polymesh.ts

```diff
@@ -547,8 +560,12 @@ export class Polymesh {
    * @param args.owner - identity representation or Identity ID as stored in the blockchain
    *
    * @note tokens with unreadable characters in their tickers will be left out
+   * @note supports pagination
    */
-  public async getSecurityTokens(args?: { owner: string | Identity }): Promise<SecurityToken[]> {
+  public async getSecurityTokens(args?: {
```

Comment:
Can't add pagination to this one because results are filtered on the client side